explode/rpm.go

```diff
@@ -17,7 +17,7 @@
 package explode
 
 import (
-	"io"
+	"os"
```

Comment:
The import list should be sorted alphabetically.

Reply:
will re-edit, was not aware of the rule.

> @jurobystricky I just tested `abireport scan-packages ...` with these changes, and it seems the program hangs for rpms that contain no files. There are many rpms shipped in Clear Linux OS that contain no files, so it's important we don't break that case. Can you reproduce the issue for this case?

I will try to reproduce. I tested the opposite, mostly huge RPMs.  Interesting, this should not depend on pipe type, a pipe is a pipe. Can you recommend a failing RPM?

Reply:
Yes, try it with https://cdn.download.clearlinux.org/releases/32610/clear/x86_64/os/Packages/m4-1.4.18-102.x86_64.rpm

Reply:
Thanks, I could reproduce the hang easily with this RPM.  BTW, I could reproduce the hang with the original unpatched abireport as well (but not as easily), just a simple loop (few 100 iterations taking less than a minute) running:
<pre>
abireport scan-packages  ./m4-1.4.18-102.x86_64.rpm
</pre>

Reply:
Ah, it took 1294 iterations for me to reproduce, but I see the hang now :-) Thanks.